lib/rules/eol-last.js

```diff
@@ -1,62 +1,95 @@
 /**
- * @fileoverview Require file to end with single newline.
+ * @fileoverview Require or disallow file to end with single newline.
  * @author Nodeca Team <https://github.com/nodeca>
+ * @author kdex <https://github.com/kdex>
```

Comment:
We reserve the `@author` field for the file creator - your contribution will be in the Git history though!